server/routes/ssot.js

```diff
@@ -19,30 +18,22 @@ router.get(
   retrievalController.getRobotList
 );
 router.get('/renameRobot', retrievalController.renameRobot);
-router.get(
-  '/retrieveMetadataForRobot/:robotId',
-  retrievalController.retrieveRobotMetadata
-);
-router.get('/shareRobotWithUser', retrievalController.shareRobotWithUser); // not used
+router.get('/shareRobotWithUser', retrievalController.shareRobotWithUser); // needed in the future
```

Comment:
I guess the comment is good to know, but I am not sure if we should have it in here, as all the other comments should be gone now too. I'd leave this up to you

Reply:
I forgot to delete it. I have deleted it now ✅